config/default-test.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
+const _ = require('lodash');
 const path = require('path');
 
 const defaultCfg = require('./default');
 
+const config = _.cloneDeep(defaultCfg);
```

Comment:
Why are you cloning this? The `config` module will deep merge these together, so we should only need to override the specific object we want to change